### PR TITLE
kubernetes-cli 1.8.2

### PR DESCRIPTION
**Pull request by iMichka**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/kubernetes-cli.rb [hunk 1]
```diff
@@ -2,8 +2,8 @@ class KubernetesCli < Formula
   desc "Kubernetes command-line interface"
   homepage "https://kubernetes.io/"
   url "https://github.com/kubernetes/kubernetes.git",
```
Comment (woodruffw) [inline Formula/kubernetes-cli.rb]: Looks like they have tagged releases now, so it'd probably be good to move to those: https://github.com/kubernetes/kubernetes/archive/v1.8.2.tar.gz

Comment (cblecker) [reply Formula/kubernetes-cli.rb]: @woodruffw Can't, as the version output of the program version depends on the git tree/hash. The test of this formula uses this too.

Comment (iMichka) [reply Formula/kubernetes-cli.rb]: See also f1238c1e38981baabd66cbbb2b445cd6e4b47d7d
Though the commit message could be a little bit more explicit :)

Comment (woodruffw) [reply Formula/kubernetes-cli.rb]: Cool; thanks for the explanation @cblecker and @iMichka!

**Pull request closed without merging**